airbyte-integrations/bases/source-acceptance-test/setup.py

```diff
@@ -2,11 +2,19 @@
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
 
+import os
 
 import setuptools
 
+CDK_VERSION = "airbyte-cdk~=0.2"
+
+if os.getenv("USE_LOCAL_CDK"):
```

Comment:
Could we only rely on `LOCAL_CDK_DIR` from there on? I fear that `USE_LOCAL_CDK` is doing two things currently:
* Indicate if we should build the docker images with the proper information
* Indicate if pip should install the local version or not

I guess maybe one way to answer this is to ask: 
* Running `pip install -e .` on the `airbyte-integrations/bases/source-acceptance-test`, should I have to configure both `USE_LOCAL_CDK` and `LOCAL_CDK_DIR` in order to build with the local version of airbyte-cdk? 
* What should happen is one is defined and not the other?

Reply:
This is definitely something that I wanted to circle back to - I'm working on cleaning it up. The current issue with using just `LOCAL_CDK_DIR` is that it's always defined in the base image. However, that also means that the images that inherit from the base image always have `LOCAL_CDK_DIR` set, even if the user wanted to run the code against the pypi package.

The fix that I'm working on is to only copy the local CDK to the base image if `LOCAL_CDK_DIR` is defined.

Reply:
Resolving this as the updates will only require a user to specify one environment variable.